x/delegation/keeper/delegation_state.go

```diff
@@ -40,68 +38,76 @@ func (k Keeper) UpdateStakerDelegationTotalAmount(ctx sdk.Context, stakerID stri
 }
 
 // GetStakerDelegationTotalAmount query the total delegation amount of the specified staker and asset.
-func (k Keeper) GetStakerDelegationTotalAmount(ctx sdk.Context, stakerID string, assetID string) (opAmount sdkmath.Int, err error) {
-	c := sdk.UnwrapSDKContext(ctx)
-	store := prefix.NewStore(c.KVStore(k.storeKey), delegationtype.KeyPrefixRestakerDelegationInfo)
+func (k *Keeper) GetStakerDelegationTotalAmount(ctx sdk.Context, stakerID string, assetID string) (opAmount sdkmath.Int, err error) {
+	store := prefix.NewStore(ctx.KVStore(k.storeKey), delegationtype.KeyPrefixRestakerDelegationInfo)
 	var ret delegationtype.ValueField
-	prefixKey := types.GetAssetStateKey(stakerID, assetID)
-	if !store.Has(prefixKey) {
+	prefixKey := stakingtypes.GetJoinedStoreKey(stakerID, assetID)
+	isExit := store.Has(prefixKey)
+	if !isExit {
```

Comment:
Remove `isExit` since it is not used again.
```suggestion
	if !store.Has(prefixKey)
```

Reply:
Thanks